recipes/lanceotron/meta.yaml

```diff
@@ -0,0 +1,64 @@
+{% set name = "LanceOtron" %}
+{% set version = "1.2.7" %}
+{% set sha256 = "2d33ec07f8f7038b6f6a34582329de6a1cdd8a5634f123ea18bbe2cef10da7b7" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://github.com/LHentges/LanceOtron/archive/36cdafa6ba697b0ced022c1344d7ead8e11c6272.tar.gz
```

Comment:
Can you please use a release version and not a commit?

Reply:
I have asked the dev to make a release, but  now we can just use the commit...

Reply:
why not use the releases from PyPI? https://pypi.org/project/lanceotron